src/server/formations-initiales-detail/useCases/consulterDetailFormationInitiale.useCase.ts

```diff
@@ -0,0 +1,20 @@
+import { CmsRepository } from '~/server/cms/domain/cms.repository';
+import { createSuccess, Either, isFailure } from '~/server/errors/either';
+import { FormationInitialeRepository } from '~/server/formations-initiales/domain/formationInitiale.repository';
+import { FormationInitialeDetailComplete } from '~/server/formations-initiales-detail/domain/formationInitiale';
+
+export class ConsulterDetailFormationInitialeUseCase {
+	constructor(private readonly formationInitialeRepository: FormationInitialeRepository, private readonly cmsRepository: CmsRepository) {
+	}
+
+	async handle(id: string): Promise<Either<FormationInitialeDetailComplete>> {
+		const formationInitialeDetail = await this.formationInitialeRepository.getDetail(id);
+		const formationInitialeDetailCMS = await this.cmsRepository.getFormationInitialeById(id);
```

Comment:
Dans la même veine que la suggestion sur le type : 

```suggestion
		const formationInitiale = await this.formationInitialeRepository.getFormationInitiale(id);
		const detailsFormationInitiale = await this.cmsRepository.getFormationInitialeById(id);
```